UnityProject/Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -129,6 +168,7 @@ public static void Play(string name, float volume, float pitch = -1, float time
 	/// </summary>
 	public static void Play(string name)
```

Comment:
Could you add an example to the summary comments, showing how to use the wildcard to play a random sound